tests/keras/engine/test_training.py

```diff
@@ -30,54 +28,56 @@ def __len__(self):
         return self.sequence_length
 
     def __getitem__(self, idx):
-        return [np.random.random((self.batch_size, 3)), np.random.random((self.batch_size, 3))], [
-            np.random.random((self.batch_size, 4)),
-            np.random.random((self.batch_size, 3))]
+        return ([np.random.random((self.batch_size, 3)),
+                 np.random.random((self.batch_size, 3))],
+                [np.random.random((self.batch_size, 4)),
+                 np.random.random((self.batch_size, 3))])
 
     def on_epoch_end(self):
         pass
 
 
 @keras_test
-def test_check_array_lengths():
-    _check_array_lengths(None, None, None)
+def test_check_array_length_consistency():
+    training_utils.check_array_length_consistency(None, None, None)
     a_np = np.random.random((4, 3, 3))
-    _check_array_lengths(a_np, a_np, a_np)
-    _check_array_lengths([a_np, a_np], [a_np, a_np], [a_np, a_np])
-    _check_array_lengths([None], [None], [None])
+    training_utils.check_array_length_consistency(a_np, a_np, a_np)
+    training_utils.check_array_length_consistency(
+        [a_np, a_np], [a_np, a_np], [a_np, a_np])
+    training_utils.check_array_length_consistency([None], [None], [None])
 
     b_np = np.random.random((3, 4))
     with pytest.raises(ValueError):
-        _check_array_lengths(a_np, None, None)
+        training_utils.check_array_length_consistency(a_np, None, None)
     with pytest.raises(ValueError):
-        _check_array_lengths(a_np, a_np, None)
+        training_utils.check_array_length_consistency(a_np, a_np, None)
     with pytest.raises(ValueError):
-        _check_array_lengths([a_np], [None], None)
+        training_utils.check_array_length_consistency([a_np], [None], None)
     with pytest.raises(ValueError):
-        _check_array_lengths([a_np], [b_np], None)
+        training_utils.check_array_length_consistency([a_np], [b_np], None)
     with pytest.raises(ValueError):
-        _check_array_lengths([a_np], None, [b_np])
+        training_utils.check_array_length_consistency([a_np], None, [b_np])
 
 
 @keras_test
-def test_slice_arrays():
+def testslice_arrays():
```

Comment:
please rename to test_slice_arrays()

Reply:
less freedom - keras-team/keras change. Since minor change avoiding merge conflicts